src/Rules/StandardRule.php

```diff
@@ -0,0 +1,17 @@
+<?php
+
+namespace GOL\Rules;
+
+use GOL\Boards\Field;
+
+class StandardRule extends Rule

```

Comment:
Please document this class!
And why does this not have an implementation for description()?